exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,31 @@
   Update the code so that you get the expected result.
 */
 
```

Comment:
well done! I think if we put a meaningful word as a parameter or variable name , our code will be more readable.